cypress/commons/actions/generic/Login.js

```diff
@@ -29,25 +24,17 @@ function login() {
     },
   });
   Login.getMicrosoftLoginButton().click();
-
   api.waitAlias(reqAuthAlias);
-  api.waitAlias(reqGetScenariosAlias, { timeout: 60 * 1000 });
-  api.waitAlias(reqGetDatasetsAlias, { timeout: 60 * 1000 });
-  api.waitAlias(reqGetWorkspaceAlias, { timeout: 60 * 1000 });
-  api.waitAlias(reqGetSolutionAlias, { timeout: 60 * 1000 });
-  api.waitAlias(reqPowerBIAlias);
+  api.waitNewPageQueries(newPageQueries);
   Scenarios.getScenarioViewTab(60).should('be.visible');
   cy.saveLocalStorage();
 }
 
 function relogin() {
```

Comment:
Could we add a `url` parameter for `relogin` too ?

Reply:
Good idea !